proto/iotics/api/meta.proto

```diff
@@ -23,25 +23,23 @@ option php_namespace = "Iotics\\Api";
 // Services only affect local resources, unless stated otherwise.
 service MetaAPI {
 
-  // SparqlQuery performs a SPARQL 1.1 query and returns one or more results, each as a sequence of chunks. Note that:
-  // - Chunks for a particular result will arrive in-order though they might be interleaved with chunks from other
-  //   results (when performing a non-local query). See scope parameter in SparqlQueryRequest;
-  // - The call will only complete once the (specified or host default) request timeout has been reached. The client can
-  //   choose to end the stream early once they have received enough results. (E.g. in the case of Scope.LOCAL this
-  //   would be after the one and only sequence of chunks has been received.). (local and remote)
-  rpc SparqlQuery(SparqlQueryRequest) returns (stream SparqlQueryResponse) {}
+  // FKGQuery performs a SPARQL 1.1 query against the Federated Knowledge Graph of the Iotics network to which this host
+  // belongs. The result is returned as a sequence of chunks. Note that:
+  // - Result chunks MIGHT arrive out of order and it is the client's responsibility to re-assemble them.
+  // - This RPC is currently in alpha!
+  rpc FKGQuery(FKGQueryRequest) returns (stream FKGQueryResponse) {}
```

Comment:
What do think of just keeping the `SparqlQuery` naming? 
Seems like `FKGQuery` just muddies the water.
The doc string can indicate the enhanced abilities, but its still just sending a sparql query?

Reply:
Sure - I hadn't realised we were happy with keeping exactly the same name :+1:

Reply:
24d01b6